src/main/groovy/com/pszymczyk/consul/ConsulStarter.groovy

```diff
@@ -51,7 +53,10 @@ class ConsulStarter {
         }
 
         ConsulPorts consulPorts = new ConsulPorts(httpPort)
```

Comment:
Ok i see it now, I think it's good approach to create two files and pass whole directory.